package.json

```diff
@@ -52,6 +52,7 @@
   },
   "scripts": {
     "test-bare": "node bin/test",
+    "test-isolated": "node bin/test-isolated",
```

Comment:
Should this be our default? Is there a need to keep the `test-bare`? / `test`?

Reply:
It's slower, but I would definitely run it in one Node.js version run, and I planned to configure that together with migration of Travis.yml to stages (which we agreed to pursue after https://github.com/serverless/serverless/pull/6132 is in).

Concerning `test-bare`. I believe we need to have command where we run tests but with no coverage report. It's faster and doesn't mangle the stack traces.

I would also run coverage report just once in CI, and in all other cases pursue regular test.

We may rename scripts so `test` is just test and `coverage` is test with coverage.
But I would also leave it for .travis.yml conversion PR